app/templates/events/event_list.html

```diff
@@ -1,8 +1,122 @@
-<h1>Event List</h1>
-
-<ul>
-    {{user}}
-    {% for event in events %}
-        <li>{{event}} </li>
-    {% endfor %}
-</ul>
+{% extends "base.html"%}
+{% block scripts %}
+  {{super()}}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/js/bootstrap-select.js"></script>
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+  </script>
+{% endblock %}
+{% block styles %}
+  {{super()}}
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/css/bootstrap-select.css" />
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+{% endblock %}
+{% block app_content %}
+
+<h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
+<div class="container-fluid px-1 py-3">
+  <div class="row d-flex align-items-baseline p-3">
+    <div class="col-md-1 ps-1 ms-auto">
+      <h4><strong>Term</strong></h4>
+    </div>
+    <div class="col-md-3 me-auto">
+      <div class="form-group">
+        <select class="form-control" name='eventTerm' id='inputTerm' onchange="location = this.value;">
+          {% for term in listOfTerms %}
+            <option value='/events/{{term}}/' {{ "selected" if selectedTerm.id == term.id }}>{{term.description}} </option>
+          {% endfor %}
+        </select>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div>
+  <ul class="nav nav-pills nav-fill mx-3 mb-3" id="pills-tab" role="tablist">
+  <li class="nav-item" role="presentation">
+    <button class="nav-link active" id="studentLedEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-student-led" type="button" role="tab" aria-controls="pills-student-led" aria-selected="true">Student Led Service</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="trainingEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-training" type="button" role="tab" aria-controls="pills-training" aria-selected="false">Training and Education</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="bonnerScholarsEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-bonner-scholars" type="button" role="tab" aria-controls="pills-bonner-scholars" aria-selected="false">Bonner Scholars</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="oneTimeEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-one-time" type="button" role="tab" aria-controls="pills-one-time" aria-selected="false">One-time Events</button>
+  </li>
+  </ul>
+</div>
+{% macro createAccordion(categoryPrograms) %}

```

Comment:
this is old, which makes me think you need to update this pr with the latest development changes